### ISSUE TITLE
BC Wallet Variation Build

### ISSUE THREAD

**Issue by cvarjao** [bcgov/bc-wallet-mobile#15]
# TL;DR

Our goal is to develeope a wallet app capable of holding verifiable credentials that is based on Aries Bifold.

We will base our work of Aries Bifold so that we may leverage the collective comunity development effort to reduce the work of any single team.

We will default to the most simple solution that allows us to acomplish our goals with no imediat long term issues. When the solution no longer works, we will make small iterative changes to overcome said issue as they arise.

# Intial Solution

Our initial requirements include a release of Aries Bifold, AS-IS, to the Google Play and Apple iTunes store with the following customizations:

- BC Gov colour palette in place; and
- BCSans Font; and
- Bundle / Package IDs set to `ca.bc.gov.BCWalelt`; and
- Applicaiton icons.

To meet our initial requriements we will:

1. Repository Management

We're not sure if we shoulf fork or just push existing bifold code to our BC Wallet repository. Forking would require us to re-create all our issues and cannot be broken easily in the future. 

For the time being we'll continue with our seporate repo with matching history.

2. Implement 

We will implement the 4 requirements above and have apply them as a patch during the build process. They will not be a permanent change commited to the up-stream (Hyperledger) repo.

When this strategy is no longer viable we will iterate to the next least complicated and sustanable strategy to solve our issue.

3. Upstream

We will continiously pull up-stream changes into our "fork" so that we are in sync.

4. The Build

There is quite a bit of work to solve "the last mile" distribution of the BC Wallet. 

- We need solve application signing in a secure and reliable way so that the build process can produce an artifact accepted by Apple and Google.

- We need to solve automated upload to both iTunes and Google Play so the signed artifact can be distrubted to testers or released for public consumtpion.

Give we're doing infrequent builds and its not overly complicated to do we'll adopt the same process we used for the BC Smart Health Card Wallet PoC: Manual builds on developer laptops. This will let us solve our imediate problem now, while working on the fully automated sultion as needed.

Acceptance Criteria:
- There is a build and deployment pipeline in place
- A few selected people may download the app from the Apple App Store
- A few selected people may download the app from the Google App Store
